src/controllers/PartialUserFormVerifyController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+
+namespace Firesphere\PartialUserforms\Controllers;
+
+use Firesphere\PartialUserforms\Forms\PasswordForm;
+use Firesphere\PartialUserforms\Models\PartialFormSubmission;
+use Page;
+use PageController;
+use SilverStripe\Control\HTTPResponse;
+use SilverStripe\Control\HTTPResponse_Exception;
+use SilverStripe\UserForms\Model\UserDefinedForm;
+
+/**
+ * Class \Firesphere\PartialUserforms\Controllers\PartialUserFormVerifyController
+ *
+ */
+class PartialUserFormVerifyController extends PageController
+{
+
+    /**
+     * @var array
+     */
+    private static $allowed_actions = [
+        'getForm'
+    ];
+    /**
+     * @var PartialFormSubmission
+     */
+    protected $partialFormSubmission;
+
+    /**
+     * @var PasswordForm
+     */
+    protected $form;
+
+    /**
+     * @return PartialUserFormVerifyController|void
+     * @throws HTTPResponse_Exception
+     */
+    public function init()
+    {
+        parent::init();
+        $session = $this->getRequest()->getSession();
+        $sessionKey = PartialSubmissionController::SESSION_KEY;
```

Comment:
I actually meant something like this
```suggestion
        $submissionID = $session->get(PartialSubmissionController::SESSION_KEY);
        // Set the session if the last session has expired
        if (!$submissionID) {
            return $this->httpError(404);
        }
        /** @var PartialFormSubmission $partial */
        $partial = PartialFormSubmission::get()->byID($submissionID);
```

Reply:
Doesn't make sense assigning a const to a variable, also do you want to include changing failover in this PR instead?

Reply:
@Firesphere rebase please

Reply:
Yaps, and also, yaps... my head is not really there

Reply:
Fixed up

Reply:
I'm not sure about the failover, as this controller uses init, it would possibly go entirely haywire with an endless loop